### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by yaoandy107**
## Summary 
Add feature's and bug's issue template

## Todos
- [x] Add template of issue/bug

## Issues
none

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
```
Comment (freddy5566) [inline .github/ISSUE_TEMPLATE/bug_report.md:34]: do we really need browser selection?

Comment (yaoandy107) [reply .github/ISSUE_TEMPLATE/bug_report.md:34]: Maybe in the future?

Comment (freddy5566) [reply .github/ISSUE_TEMPLATE/bug_report.md:34]: you wanna develop web application in flutter?

Comment (yaoandy107) [reply .github/ISSUE_TEMPLATE/bug_report.md:34]: It looks easy to support the web in the Flutter project. Why don't you give Flutter web a chance?

**Review by freddy5566 — APPROVED** [review]

**Pull request merged**